core/client/fs/src/main/java/alluxio/client/block/stream/BlockWorkerClient.java

```diff
@@ -172,4 +174,11 @@ StreamObserver<OpenLocalBlockRequest> openLocalBlock(
    * @return status
    */
   GetStatusPResponse getStatus(GetStatusPRequest request);
+
+  /**
+   * List status from Worker.
+   * @param request
+   * @return List of Status
+   */
+  java.util.Iterator<ListStatusPResponse> listStatus(ListStatusPRequest request);
```

Comment:
Add an import for `java.util.Iterator`

Reply:
Fixed.

importing packages, or using the full package names, are the two ways to reference a package outside of this package.
Do we have a coding style requirement to "import a package" instead of "use full package name"?

Reply:
I think it's "always import the symbol except when it's ambiguous. Then use the fully qualified name." Like what we do with `alluxio.grpc.FileInfo` and `alluxio.wire.FileInfo`.

Reply:
Got it.